tfe/resource_tfe_workspace.go

```diff
@@ -250,11 +261,23 @@ func resourceTFEWorkspaceCreate(d *schema.ResourceData, meta interface{}) error
 		options.TerraformVersion = tfe.String(tfVersion.(string))
 	}
 
-	if tps, ok := d.GetOk("trigger_prefixes"); ok {
-		for _, tp := range tps.([]interface{}) {
-			if t, ok := tp.(string); ok {
-				options.TriggerPrefixes = append(options.TriggerPrefixes, t)
+	if d.HasChange("trigger_prefixes") {
```

Comment:
This `HasChange` is needed in create function?

Reply:
Yeah good point but we need https://github.com/hashicorp/go-tfe/pull/410/files merged for that